stories/modal.stories.js

```diff
@@ -0,0 +1,230 @@
+          Show Modal
+        </button>
+        <div
+          className="Manifold__Modal"
+          role="dialog"
+          aria-modal={isModalVisible || undefined}
+          aria-labelledby="dialog-title"
+          aria-describedby="dialog-desc"
+          onClick={() => closeModal()}
+        >
+          <div onClick={(e) => e.stopPropagation()}>
+            <button
+              type="button"
+              className="Manifold__Modal__Close"
+              onClick={() => closeModal()}
+              aria-label="Close"
+            >
+              ✕
+            </button>
+            <h1 id="dialog-title">Modal Title</h1>
+            <p id="dialog-desc">This is a description describing the modal.</p>
+            <p>
+              Babybel camembert de normandie lancashire. Say cheese cauliflower cheese danish
+              fontina say cheese cheesy feet bocconcini dolcelatte fromage frais. Cheese slices
+              fondue cheese triangles bavarian bergkase queso emmental cream cheese manchego.
+              Port-salut cottage cheese smelly cheese chalk and cheese boursin feta macaroni cheese
+              camembert de normandie. Macaroni cheese.
+            </p>
+            <p>
+              Cheese on toast cheddar cheesecake. Mozzarella cream cheese mozzarella cheese slices
+              cottage cheese cheese and wine halloumi squirty cheese. Cheese and wine camembert de
+              normandie manchego macaroni cheese croque monsieur cow rubber cheese boursin. Cheesy
+              grin stilton cheddar cheese and wine gouda ricotta.
+            </p>
+            <p>
+              Cottage cheese fromage frais emmental. Port-salut cheese strings cheese on toast
+              jarlsberg manchego cream cheese cheesy grin mascarpone. Cheese strings chalk and
+              cheese fromage frais cheesy feet blue castello macaroni cheese monterey jack melted
+              cheese. Say cheese cheeseburger goat squirty cheese fromage halloumi cow.
+            </p>
+            <p>
+              Jarlsberg halloumi bocconcini. Pecorino gouda rubber cheese emmental who moved my
+              cheese cheese and wine airedale melted cheese. Cauliflower cheese hard cheese croque
+              monsieur parmesan cauliflower cheese babybel gouda pepper jack. Cut the cheese
+              port-salut cream cheese lancashire caerphilly gouda fromage frais.
+            </p>
+            <p>
+              Roquefort say cheese boursin. Emmental gouda emmental st. agur blue cheese smelly
+              cheese rubber cheese fondue squirty cheese. Hard cheese cut the cheese cheeseburger
+              boursin taleggio port-salut edam red leicester. Squirty cheese brie cheesy feet
+              ricotta who moved my cheese cottage cheese rubber cheese airedale. Swiss edam cut the
+              cheese croque monsieur.
+            </p>
+            <footer className="Manifold__Modal__Footer">
```

Comment:
💯 I would actually love to implement a pattern here, like right-side primary action (we’ve been inconsistent about that)! I just didn’t, here, to show that anything could go in the footer, but maybe we _shouldn’t_ allow anything in the footer and should encourage certain paradigms.

Let me do some research on the best way to do that accessibly (it’s fun because in a form, first button in HTML takes priority so you have to reverse the ordering in CSS, but that also has unexpected consequences for screen readers)

Reply:
Awesome, thanks. I do think we should keep it consistent and encourage the paradigm.

Reply:
Gonna resolve this comment to clean up, but feel free to leave another for feedback!